tests/export-graph.sh

```diff
@@ -22,9 +22,18 @@ nix-store --gc # should force rebuild of input-1
 
 outPath=$(nix-build ./export-graph.nix -A 'foo."bar.buildGraph"' -o $TEST_ROOT/result)
 
-checkRef input-1
 checkRef input-1.drv
-checkRef input-2
```

Comment:
@Ericson2314 Why had I to remove these two checks? This test should build also outputs even after removing that piece of code, right?